APMrover2/sailboat.h

```diff
@@ -33,7 +34,7 @@ class Sailboat
     void init();
 
     // update mainsail's desired angle based on wind speed and direction and desired speed (in m/s)
-    void  update_mainsail(float desired_speed);
+    float update_sail_control(float desired_speed, float throttle_out);
```

Comment:
we should add a comment saying what is being returned..

Reply:
fixed